satellitepy/data/tools.py

```diff
@@ -112,20 +113,24 @@ def save_patches(
     rescaling : float
         Images and the corresponding labels will be rescaled.
     interpolation_method : str
-        <interpolation-method> will be used to rescale images. 
+        <interpolation-method> will be used to rescale images.
+    keep_patches : bool
+        Empty patches are kept if true.
     Returns
     -------
     Save patches in <out-folder>/images and <out-folder>/labels
     """
     logger.info('Saving patches')
 
+    assert out_folder.exists(), 'Make sure the out-folder exists or is created before calling this function.'
+
     out_image_folder = Path(out_folder) / 'images'
     out_label_folder = Path(out_folder) / 'labels'
 
-    assert create_folder(out_image_folder)
+    assert create_folder(out_image_folder, ask_permission=False)
```

Comment:
Pass `ask_permission` to `args` in the initial script `tools/data/create_patches.py`. Default must be `True`.

Reply:
Default is already set to `True` in `satellitepy/utils/path_utils.py`, so for creating the folder in `tools/data/create_patches.py` you still need to agree. We use `create_folder` in a lot of places, so I didn't state `ask_permission=True` explicitly everywhere and the default behavior is unchanged.